docs/generate/generate_without_sdk_repo.md

```diff
@@ -0,0 +1,42 @@
+# <img align="center" src="../images/logo.png">  Generating C# Clients Outside of the azure-sdk-for-net Repo
```

Comment:
Is it worth to duplicate this with the main readme?

Reply:
Yeah that's a fair point, I thiink I got carried away. Will jsut make this a link to the main readme